packages/react-scripts/scripts/start.js

```diff
@@ -267,7 +306,9 @@ function runDevServer(host, port, protocol) {
       ignored: /node_modules/
     },
     // Enable HTTPS if the HTTPS environment variable is set to 'true'
-    https: protocol === "https",
+    https: (
+      protocol === 'https'
+    ),
```

Comment:
I sent a PR yesterday to fix this. The reason is that the group (the unit that determines the size of an expression to know if it should break or not) wrongly included the comment.